### PR TITLE
Feature/mms python client

### PR DESCRIPTION
**Pull request by HuiJun**

### CODE REVIEW

File: recipes/mms-python-client/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+{% set file_ext = "tar.gz" %}
+{% set hash_type = "sha256" %}
+{% set hash_value = "6855d99218ed7e219e37d6cb6628443ac3abeb1921c16e0f778b1aee3e625290" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.{{ file_ext }}'
```
Comment (chrisburr) [inline recipes/mms-python-client/meta.yaml]: This line isn't needed
```suggestion
```

File: recipes/mms-python-client/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  '{{ hash_type }}': '{{ hash_value }}'
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/mms-python-client/meta.yaml]: You probably only need `python`, `pip` and `setuptools` in the `host` requirements section.

File: recipes/mms-python-client/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+    - mms_python_client
+
+about:
+  home: http://openmbee.org
+  license: Apache 2.0
+  license_file: LICENSE
+  summary: Model Management Service
+  description: 'Model Management System (MMS) is a version control system for structured data. It exposes model information through RESTful web services that can be used for CRUD operations, branching,
+    and tagging of the model repository.'
+  doc_url: ''
```
Comment (chrisburr) [inline recipes/mms-python-client/meta.yaml]: Can you add URLs for `doc_url` and `dev_url`? If not, the entries should be removed entirely.

File: recipes/mms-python-client/meta.yaml (hunk 1 above)
Comment (HuiJun) [reply recipes/mms-python-client/meta.yaml]: Removed

File: recipes/mms-python-client/meta.yaml (hunk 2 above)
Comment (HuiJun) [reply recipes/mms-python-client/meta.yaml]: Removed everything else in the `host` section

File: recipes/mms-python-client/meta.yaml (hunk 3 above)
Comment (HuiJun) [reply recipes/mms-python-client/meta.yaml]: Removed the entries entirely.

**Comment by HuiJun** [comment]
Thanks for reviewing. Added the suggested fixes.

**Pull request merged**